my_portal_page/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module adds my portal account menu item on the top-right corner of the webclient.
```

Comment:
Please update readme.This is "Add Portal menu item in Usermenu" right?

Reply:
Okay. I will